src/SnapKeyring.test.ts

```diff
@@ -44,21 +50,41 @@ describe('SnapKeyring', () => {
 
   const snapId = 'local:snap.mock' as SnapId;
 
+  const ethEoaAccount1 = {
+    id: 'b05d918a-b37c-497a-bb28-3d15c0d56b7a',
+    address: '0xC728514Df8A7F9271f4B7a4dd2Aa6d2D723d3eE3'.toLowerCase(),
+    options: {},
+    methods: [...Object.values(EthMethod)],
+    type: EthAccountType.Eoa,
+  };
+  const ethEoaAccount2 = {
+    id: '33c96b60-2237-488e-a7bb-233576f3d22f',
+    address: '0x34b13912eAc00152bE0Cb409A301Ab8E55739e63'.toLowerCase(),
+    options: {},
+    methods: [...Object.values(EthMethod)],
+    type: EthAccountType.Eoa,
+  };
+
+  const ethErc4337Account = {
+    id: 'fc926fff-f515-4eb5-9952-720bbd9b9849',
+    address: '0x2f15b30952aebe0ed5fdbfe5bf16fb9ecdb31d9a'.toLowerCase(),
+    options: {},
+    methods: [...Object.values(EthErc4337Method)],
+    type: EthAccountType.Erc4337,
+  };
+  const btcP2wpkhAccount = {
+    id: '11cffca0-12cc-4779-8f82-23273c062e29',
+    address: '1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa'.toLowerCase(),
```

Comment:
Its satoshi's address, i think its the legacy address format. I agree, we should be more restrictive with the address. It would require an update in the keyring-api